test/sagemaker_tests/pytorch/training/conftest.py

```diff
@@ -493,6 +493,68 @@ def skip_s3plugin_test(request):
             )
 
 
+@pytest.fixture(autouse=True)
+def skip_smdebug_test(
+    request,
+    processor,
+    ecr_image,
+):
+    if request.node.get_closest_marker("skip_smdebug_test"):
+        _, image_framework_version = get_framework_and_version_from_tag(ecr_image)
+        image_cuda_version = get_cuda_version_from_tag(ecr_image) if processor == "gpu" else ""
+        if (
+            Version(image_framework_version) in SpecifierSet("==2.0.1")
```

Comment:
This looks like a pretty strict condition. Is it safe to specify btwn 2.0 and 2.1, in case we release patch versions like 2.0.2 in the future?

Can update such that `SpecifierSet("==2.0.*")`

Comment applies to other instances of this as well